rpm/sofia-sip.spec

```diff
@@ -23,59 +26,59 @@ multimedia distribution, and multimedia conferences.
 
 
 %package glib
-Summary:    Glib bindings for Sofia-SIP
-Group:      Communications/Telephony and IM
+Summary:    Glib bindings for Sofia-SIP 
 Requires:   %{name} = %{version}-%{release}
-Requires(post): /sbin/ldconfig
-Requires(postun): /sbin/ldconfig
 
 %description glib
 GLib interface to Sofia SIP User Agent library.
 
 %package utils
 Summary:    Sofia-SIP Command Line Utilities
-Group:      Applications/Internet
 Requires:   %{name} = %{version}-%{release}
 
 %description utils
 Command line utilities for the Sofia SIP UA library.
 
 %package glib-devel
 Summary:    Glib bindings for Sofia SIP development files
-Group:      Development/Libraries
-Requires:   %{name} = %{version}-%{release}
-Requires:   sofia-sip-glib = %{version}-%{release}
-Requires:   sofia-sip-devel = %{version}-%{release}
+Requires:   %{name}-glib = %{version}-%{release}
+Requires:   %{name}-devel = %{version}-%{release}
+Requires:   pkgconfig
 
 %description glib-devel
 Development package for Sofia SIP UA Glib library. This package
 includes libraries and include files for developing glib programs
 using Sofia SIP.
 
-
 %package devel
 Summary:    Sofia-SIP Development Package
-Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
+Requires:   pkgconfig
 
 %description devel
 Development package for Sofia SIP UA library.
 
+%package doc
+Summary:   Documentation for %{name}
+Requires:  %{name} = %{version}-%{release}
+
+%description doc
+Documentation for %{name}.
+
+
 %prep
-%setup -q -n %{name}-%{version}/src
+%setup -q -n %{name}-%{version}/%{name}
 
 %build
-%configure --disable-static
+./autogen.sh
+%configure --disable-static --without-doxygen --disable-stun
```

Comment:
You can replace these ^ two lines with %autogen instead.

Reply:
Maybe `%reconfigure` is better.

Reply:
Couldn't find any help on `%reconfigure` macro - is it standardized? Same with `%autogen` and if all it does is `./autogen.sh` then I don't see any value in hiding that behind a macro.

Reply:
Here it is: https://github.com/sailfishos/meego-rpm-config/blob/master/macros#L61

It's better to use the same macro passing the same variables to all package builds.

Reply:
ok, so it's not standardized, indeed

Reply:
and it runs `autoreconf` twice (first time from `autogen.sh`)

Reply:
`autogen.sh` should not be called explicitly.

Reply:
> `autogen.sh` should not be called explicitly.

Why not?

Reply:
I mean, of course I looked at it and I saw that it's not doing much but since it seems to be part of their build process then so be it. This step is simply copied from their sample .spec file, I haven't invented it. I'm not a fan of the autohell stuff at all, but I'm not planning to rewrite their build system, either. It is what it is.